packages/legend-art/src/dialog/Modal.tsx

```diff
@@ -89,6 +89,7 @@ export const ModalFooterButton: React.FC<{
   className?: string;
   onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
   formatText?: boolean | undefined;
+  type?: 'primary' | 'secondary';
```

Comment:
maybe could we create a enum for this ?

Reply:
I was doing some research and it seems like there's arguments for enums and for string unions (https://www.typescriptcourse.com/string-literal-unions-over-enums). In this case, I think a string union would be better, because there's only 2 options, so the string union feels simpler and cleaner to me. Also, VS Code will still provide you with autocomplete options and highlight invalid values when you use string unions.